ACCEPTANCE_TESTS.rst

```diff
@@ -0,0 +1,163 @@
+S3 Acceptance Tests
+===================
+
+List of all of the various scenarios that need to be handled in implementing
+a S3 transfer manager.
+
+Upload Tests
+------------
+
+General
+~~~~~~~
+* [x] Upload single nonmultipart file
+* [x] Upload single multipart file
+* [x] Upload multiple nonmultipart files
+* [x] Upload multiple multipart files
+* [x] Failed/cancelled multipart upload is aborted and leaves no orphaned parts
+* [ ] Ctrl-C of any upload does not hang
```

Comment:
Not sure about an absolute value, but I would expect it would be `avg(transfer_time_iter_chunk) * margin`.  I think if we're using 8k buffer sizes for downloads, a couple of seconds should be sufficient.